internal/vfs/iovfs/iofs.go

```diff
@@ -24,6 +24,11 @@ type WritableFS interface {
 	Remove(path string) error
 }
 
+type FsWithSys interface {
+	vfs.FS
+	FSys() fs.FS
```

Comment:
I think I probably would have called this `Underlying()` but `FSys` is fine.

Reply:
Eh, it's not really "wrapping" per se, it's _adapting_ one FS to the other. Ignore this 😄